### PR TITLE
ATAC-seq

### PR DESCRIPTION
**Pull request by dpryan79**
This condafies ATAC-seq and is an example of how to use an Rscript integrated with Snakemake.

### CODE REVIEW

**Comment by dpryan79** [comment]
The only thing not in conda at the moment is `bc` (part of the QC rule). Truth be told it segfaults, so the conda-forge version seems to be broken. We could actually swap it out for python and use that from the shell, but for now I expect every system has `bc` installed.

I'd like to work on the CSAW script to update and clean it up a bit. It seems to work fine, but I'm not sure it's up to the best practice (I need to go through the user manual) and there's some extra code that can be removed. The logging is also sub-optimal at the moment.

Note that CSAW **absolutely can not** be run multithreaded. There's an option for that, but it ends up becoming a fork bomb and nearly crashes the server.

File: shared/paths.yaml [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 # 'workflow_tools' is set on runtime (see common_functions.py:load_paths())
 # paths referring to a local folder
-#R_path: "/package/R-3.3.1/bin/"
-#R_libs_path: "/data/manke/repository/scripts/snakemake_workflows/R/x86_64-unknown-linux-gnu-library/3.3/"
+#R_path: "/package/R-3.5.0/bin/"
+#R_libs_path: "/data/processing4/R_LIB/3.5/"
 #histoneHMM_path: "/data/manke/repository/scripts/snakemake_workflows/R/x86_64-redhat-linux-gnu-library/3.3/histoneHMM/bin/"
-# paths referring to /package/
+#paths referring to /package/
 #bedtools_path: "/package/bedtools2-2.25.0/bin/"
```
Comment (dpryan79) [inline shared/paths.yaml]: None of those lines are even used any more...

Comment (vivekbhr) [reply shared/paths.yaml]: LOL I don't know why I removed spaces from a commented line :D

Comment (dpryan79) [reply shared/paths.yaml]: A commented line in a file that can be deleted now...

**Pull request merged**